tests/utils_test.py

```diff
@@ -136,7 +136,7 @@ def test_resize_sequence_without_permutation2(self):
 
   def test_resize_sequence_with_permutation(self):
     """Test when we permute, each output can be one of the permutations."""
-    sub_sequence, seq_lengths, _, _ = utils.resize_sequence(
+    sub_sequence, seq_lengths = utils.resize_sequence(
         sequence=np.array([[1, 1], [2, 2], [3, 3]]),
         cluster_id=np.array([1, 2, 1]),
         num_permutations=2)
```

Comment:
Would you mind adding some simple tests for the estimate_transition_bias() function you added?

Reply:
Yes, I can. It will take some days though.

Reply:
No worries.